src/components/organisms/ChatDrawer/ChatDrawer.tsx

```diff
@@ -62,6 +63,7 @@ const ChatDrawer: React.FC<PropsType> = ({
       chatContext.sendRoomChat(user.uid, roomName, data.messageToTheBand);
     reset();
   };
+  const BEFORE_SEED = SEED_START_UTC_SECONDS - 5 * 1440000;
```

Comment:
nit: would you be able to include some justification for the choice of constant here, as well as what 1440000 is in millis